MerlinAU.sh

```diff
@@ -6388,7 +6458,7 @@ _PostUpdateEmailNotification_()
    fi
 
    _UserTraceLog_ "END of $logMsg [$$curWaitDelaySecs sec.]"
-   sleep 60  ## Let's wait a bit & proceed ##
+   sleep 30  ## Let's wait a bit & proceed ##
```

Comment:
Decrease back down now that we know this isn't the problem with the WAN connection/nodes.

Reply:
> Decrease back down now that we know this isn't the problem with the WAN connection/nodes.
>
Makes sense. In fact, I did the same thing on my PR.